bundler.d/ovirt.rb

```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'fog-ovirt', '~> 1.2.5'
+  gem 'fog-ovirt', '~> 2.0'
```

Comment:
@evgeni while we don't *have* to bump the version, leaving it at `~>1.2.5` won't allow the new 2.0.0 version. It could be changed to `>= 1.2.5.`, but we may as well update the minimum.

Reply:
That's why I said ">= 1.2.5, < 3" above ;)

Reply:
🤦🏼 that's what i get for vaguely remembering comments from mail notifications...